cni/pkg/install/install.go

```diff
@@ -26,11 +26,12 @@ import (
 	"istio.io/istio/cni/pkg/util"
 	"istio.io/istio/pkg/file"
 	"istio.io/istio/pkg/log"
-	"istio.io/istio/pkg/slices"
 )
 
 var installLog = log.RegisterScope("install", "CNI install")
 
+const istioCNIBinary = "istio-cni"
```

Comment:
This still only appears to be used in one location in code.  Why make it a global variable?